src/structures/VoiceChannel.js

```diff
@@ -12,6 +14,18 @@ let deprecationEmittedForEditable = false;
  * @extends {BaseGuildVoiceChannel}
```

Comment:
This is missing a ` * @implements {TextBasedChannel}` so the properties appear in the documentation.
Looks like I forgot this in my PR 😅